example/deploy-integration-tests-contracts.ts

```diff
@@ -1,19 +1,18 @@
 import { MichelsonMap, TezosToolkit } from '@taquito/taquito';
-import { importKey } from '@taquito/signer';
+import { InMemorySigner } from '@taquito/signer';
 import { knownContract } from './data/knownContract';
 import { knownBigMapContract } from './data/knownBigMapContract';
 import { singleSaplingStateContractJProtocol } from '../integration-tests/data/single_sapling_state_contract_jakarta_michelson';
-import Faucet from './faucet-interface';
 import { char2Bytes } from '@taquito/utils';
 import { fa2ForTokenMetadataView } from '../integration-tests/data/fa2-for-token-metadata-view';
 
-const { email, password, mnemonic, activation_code } =
-  require('./faucet-default-values.json') as Faucet;
-
 const provider = 'https://kathmandunet.ecadinfra.com/';
```

Comment:
Should this also change to ghost net?

Reply:
I left Kathmandu because I thought we use this file to deploy the known contracts used in the integration tests when there is a new protocol. So when adding support for L, we will replace Kathmandunet by Lnet. But I think we can completely delete the file because there is another script in the integration tests folder. Is it ok for you if a delete the file?

Reply:
Need to check the other script includes the contracts originated in this script

if any are missing they can be transferred over

Reply:
I will make a new issue for consolidating these two scripts into one

Reply:
Ok then I will merge this PR and we can follow up with the issue you created.